apel/db/records/storage_summary.py

```diff
@@ -0,0 +1,126 @@
+'''
+   Copyright 2012 STFC
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+'''
+
+import time
+from apel.db.records import Record
+
+import logging
+
+# get the relevant logger
+log = logging.getLogger(__name__)
+
+
+class StorageSummaryRecord(Record):
+    '''
+    Class to represent one `summarised` storage record.
```

Comment:
I don't think we need the back-ticks around "summarised", do we?

Reply:
I have used it to differentiate and stress more on `summarised` so that one can understand what it is referring to like Summarised records. It make sense that summarised alone should help us understanding it's an summarised storage records. So, I have removed the backticks.